guides/common/modules/con_transport-modes-for-remote-execution.adoc

```diff
@@ -3,41 +3,43 @@
 
 You can configure your {Project} to use two different modes of transport for remote job execution.
 
+Push-based transport::
 On {SmartProxies} in `ssh` mode, remote execution uses the SSH service to transport job details.
 This is the default transport mode.
 The SSH service must be enabled and active on the target hosts.
 The remote execution {SmartProxy} must have access to the SSH port on the target hosts.
 Unless you have a different setting, the standard SSH port is 22.
 
+Pull-based transport::
+On {SmartProxies} in `pull-mqtt` mode, remote execution uses Message Queueing Telemetry Transport (MQTT) to publish jobs it receives from {ProjectServer}.
+The host subscribes to the MQTT broker on {SmartProxy} for job notifications using the `yggdrasil` pull client.
+After the host receives a notification, it pulls job details from {SmartProxy} over HTTPS, runs the job, and reports results back to {SmartProxy}.
++
+To use the `pull-mqtt` mode, you must enable it on {SmartProxyServer} and configure the pull client on hosts.
+
 [NOTE]
 ====
 If your {SmartProxy} already uses the `pull-mqtt` mode and you want to switch back to the `ssh` mode, run this `{foreman-installer}` command:
 
 [options="nowrap",subs="+quotes,verbatim,attributes"]
 ----
-# {foreman-installer} --foreman-proxy-plugin-remote-execution-script-mode=ssh
+# {foreman-installer} --foreman-proxy-plugin-remote-execution-script-mode ssh
 ----
 ====
 
-On {SmartProxies} in `pull-mqtt` mode, remote execution uses Message Queueing Telemetry Transport (MQTT) to publish jobs it receives from {ProjectServer}.
-The host subscribes to the MQTT broker on {SmartProxy} for job notifications using the `yggdrasil` pull client.
-After the host receives a notification, it pulls job details from {SmartProxy} over HTTPS, runs the job, and reports results back to {SmartProxy}.
-
-To use the `pull-mqtt` mode, you must enable it on {SmartProxyServer} and configure the pull client on the target hosts.
-
 .Additional resources
-* To enable pull mode on {SmartProxyServer}, see {InstallingSmartProxyDocURL}configuring-remote-execution-for-pull-client_{smart-proxy-context}[Configuring Remote Execution for Pull Client] in _{InstallingSmartProxyDocTitle}_.
+* To enable pull mode on {SmartProxyServer}, see {InstallingSmartProxyDocURL}configuring-remote-execution-for-pull-client_{smart-proxy-context}[Configuring pull-based transport for remote execution] in _{InstallingSmartProxyDocTitle}_.
 ifdef::katello,orcharhino,satellite[]
-* To enable pull mode on an existing host, continue with xref:Configuring_a_Host_to_Use_the_Pull_Client_{context}[].
+* To enable pull mode on a registered host, continue with xref:Configuring_a_Host_to_Use_the_Pull_Client_{context}[].
```

Comment:
I'm not sure about this. In my mind, registered is a subset of existing and that doesn't feel right. For example the foreman machine itself is not registered, but you can still configure pull on it

Reply:
How about just "a host"? Or I can just keep the original if you think that it's be too generic.

Reply:
What does it mean that "a host is registered" anyway? The Foreman server does appear in the All Hosts index by default, therefore, it is registered. No?

Reply:
Also, in the Prerequisites of the xref'd section, we say that the host must be registered.

Reply:
I'll just say "a host". But there's still the prerequisite :D